File: datasets/IR-Plag/convertai-original-java-rest-python/case-06/plagiarized/L6/01/L6.py
import sys

def print_array(arr):
    i = 9
    j = 0
    while j < 10:
        print(arr[i])
        i -= 1
        j += 1

if __name__ == "__main__":
    angka = [0] * 10
    i = 0
    while i != 10:
        print("Read a number: ", end="")
        angka[i] = int(input())
        i += 1

    print_array(angka)

File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-06/plagiarized/L6/01/L6.ts
import * as readline from 'readline';

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

function print(arr: number[]) {
  let i = 9;
  let j = 0;
  while (j < 10) {
    console.log(arr[i]);
    i--;
    j++;
  }
}

function main() {
  const angka: number[] = new Array(10);
  let i = 0;
  while (i !== 10) {
    rl.question('Read a number: ', (input) => {
      angka[i] = parseInt(input);
      i++;
      if (i === 10) {
        print(angka);
        rl.close();
      }
    });
  }
}

main();

